compact/pid/dirc.xml

```diff
@@ -94,8 +94,11 @@
   </regions>
 
   <display>
-    <vis name="DIRCTube"   ref="AnlTeal" alpha="0.1" visible="true"  showDaughters="true"   lineStyle="solid" drawingStyle="solid" />
-    <vis name="DIRCBox"    ref="AnlLight_Gray" alpha="0.1" visible="true"  showDaughters="true"   lineStyle="solid" drawingStyle="solid" />
+    <!-- Commented versions while bug in DD4Hep overwites alpha value of ref -->
+    <!--vis name="DIRCTube"   ref="AnlTeal" alpha="0.1" visible="true"  showDaughters="true"   lineStyle="solid" drawingStyle="solid" /-->
+    <!--vis name="DIRCBox"    ref="AnlLight_Gray" alpha="0.1" visible="true"  showDaughters="true"   lineStyle="solid" drawingStyle="solid" /-->
```

Comment:
Link to DD4hep bug report, and postpone these changes until fixed, if still needed then.